website/docs/docs-section/faq.md

```diff
@@ -6,22 +6,22 @@ sidebar_label: FAQ
 
 #### What are OpenAttestation documents?
 
-OpenAttestation documents are verifiable documents, or Verifiable Credentials, as commonly referred to by the tech community. 
+OpenAttestation documents (OA documents) are also known as verifiable documents or verifiable credentials, as the technical community commonly refers it to. 
```

Comment:
The reason is the active voice is preferred in formal writing, instead of the passive voice. 
So I was trying to change it to active voice without changing the meaning... 

If verifiable credentials are different from verifiable documents, then I suggest deleting "or Verifiable Credentials" here, and adding a new entry for it in "Glossary" with the link you provided.

Reply:
The current version is incorrect as OA docs are not also known as verifiable documents. They are verifiable documents but we do not use the terms interchangeably. 

Let's just go with `OpenAttestation documents (OA documents) are a kind verifiable documents or verifiable credentials.` or something to this effect, without the "refers it to" part.

Reply:
updated to: 

> OpenAttestation documents (OA documents) are a type of verifiable documents or verifiable credentials.

Reply:
also updated "Glossary"